modules/web-console/e2e/testcafe/fixtures/help-menu.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2019 GridGain Systems, Inc. and Contributors.
+ *
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {dropTestDB, insertTestUser, resolveUrl} from '../environment/envtools';
+import {createRegularUser} from '../roles';
+import * as helpMenu from '../components/helpMenu';
+import {userMenu} from '../components/userMenu';
+import * as gettingStarted from '../components/gettingStartedDialog';
+
+const user = createRegularUser();
+
+fixture('Help menu')
+    .beforeEach(async(t) => {
+        await dropTestDB();
+        await insertTestUser();
+        await t.useRole(user);
+    })
+    .afterEach(async(t) => {
+        await dropTestDB();
+    });
+
+test.page(resolveUrl('/'))('Help menu items', async(t) => {
+    await t
+        .hover(helpMenu.trigger)
+        .expect(helpMenu.item('Documentation').getAttribute('href')).eql('https://docs.gridgain.com/docs/web-console')
```

Comment:
Possibly we should also check that target attribute equals "_blank"

Reply:
Good point, I'll add this.